src/ORM/Versioning/ChangeSet.php

```diff
@@ -535,6 +541,20 @@ public function getDescription()
         return $string;
     }
 
+    /**
+     * Get number of changes
+     *
+     * @return int
+     */
+    public function getChangesCount()
```

Comment:
Why isn't `Changes()` always returning an empty list, like other ORM relations? In that case the method wouldn't be needed